### DISCUSSION TITLE
Interpretations regarding the covariance between random intercepts

### DISCUSSION THREAD

**Discussion by sojata** [cdriveraus/ctsem#51] [Q&A] [answered]
Hi All,

I have played around with the CT-VAR model concerning depression and anxiety symptoms during the psychotherapeutic treatment in a large sample (6000>) with four measurements. I estimate first-order auto- and cross-effects between the symptoms, intercepts, T0 means, diffusion variances/covariance, measurement errors for symptoms, and effects of progress in the treatment session (time-variant predictor) on symptoms in the model. Moreover, I model random effects for T0 means and intercepts to capture stable individual differences in these parameters. Find attached the matrix description of the model.

I struggle a bit with interpreting the rawpopcorr results in the model output. I found strong correlations between all T0 means and intercepts. Can the positive correlations between random intercepts be interpreted as a between-person correlation between the baseline levels of depression and anxiety or do they reflect more the correlation between some kind of accumulative level of the symptoms? Or something else? Specifically, does this interpretation holds for the correlated random intercepts: People that show higher levels of depression during the treatment also, on average, show higher levels of anxiety during the treatment compared to people with lower levels of depression. Similarly, does the positive correlation e.g., between T0 means and intercept of depression indicate that people showing higher levels of depression at the beginning also (on average) show higher levels across the treatment compared to those showing lower levels of depression at the beginning?

I realize that in these models, intercept defines the baseline level of the processes together with the drift matrix, but I yet wonder about the interpretation regarding their correlation in CTSEM.

best,
Jaakko Tammilehto

[MODEL_4.pdf](https://github.com/cdriveraus/ctsem/files/12043806/MODEL_4.pdf)

**Comment by cdriveraus** [comment, marked as answer]
The model you show doesn't include the time invariant predictor, but I guess you know this? Since the t0mean captures the starting point, and the cint will generate the long term set point / equilibrium / general direction of change (if any), the positive correlation doesn't seem very surprising to me -- people who start high tend to remain high. To understand whether there is an upwards or downwards trend due to the cint is trickier because this also depends on the starting level and the temporal effects -- in some cases you may see a trend but in others you may see a simple equilibrium -- but you can see this easily if you plot using:
```
ctKalman(myfit,plot=TRUE,subjects=1,removeObs=T)
```
which shows you the model expectations conditional on estimated parameters, covariates, and timing information, but not conditioned on the individual observations (since removeObs=T)  .

**Reply by sojata** [reply]
Thank you again Charles! Yes, no TI predictors, just to take between-person cofounders into account (while keeping a model simple). Also really useful the plotting comment, now the interpretation make sense (a small downward trend).
